contrib/masterfiles-stage/README.org

```diff
@@ -232,8 +212,9 @@ Or execution can be automated with CFEngine.
 #+END_SRC
 
 If the params file is placed in the default location
-=/opt/cfengine/dc-scripts/params.sh=, and if the value of =MASTERDIR=
-is specified in the params file, the CFEngine code can be as simple as:
+=/opt/cfengine/dc-scripts/params.sh=, and masterfiles to be deployed
+are in the default location =/var/cfengine/masterfiles=, the CFEngine
```

Comment:
I think `=MASTERDIR=` was better here. When I was trying to create the packages for Fedora, I had to use a different directory for *masterfiles*.

Reply:
agreed